tests/ci/reporter_tests.js

```diff
@@ -504,18 +504,61 @@ describe('test reporters', function() {
       reporter.finish();
       var output = stream.read().toString();
 
-      assert.match(output, /##teamcity\[testSuiteFinished name='testem\.suite' duration='(\d+(\.\d+)?)'\]/);
-      assert.match(output, /##teamcity\[testStarted name='phantomjs - it does <cool> "cool" \|'cool\|' stuff']/);
-      assert.match(output, /##teamcity\[testFinished name='phantomjs - it does <cool> "cool" \|'cool\|' stuff' duration='1234']/);
-      assert.match(output, /##teamcity\[testStarted name='phantomjs - it skips stuff']/);
-      assert.match(output, /##teamcity\[testIgnored name='phantomjs - it skips stuff' message='pending']/);
-      assert.match(output, /##teamcity\[testFinished name='phantomjs - it skips stuff']/);
-      assert.match(output, /##teamcity\[testStarted name='phantomjs - it handles failures']/);
-      assert.match(output, /##teamcity\[testFailed name='phantomjs - it handles failures' message='foo' details='bar']/);
-      assert.match(output, /##teamcity\[testFinished name='phantomjs - it handles failures']/);
-      assert.match(output, /##teamcity\[testStarted name='phantomjs - it handles undefined errors']/);
-      assert.match(output, /##teamcity\[testFailed name='phantomjs - it handles undefined errors' message='' details='']/);
-      assert.match(output, /##teamcity\[testFinished name='phantomjs - it handles undefined errors' duration='42']/);
+      assert.match(output, /##teamcity\[testSuiteStarted name='phantomjs']/);
+      assert.match(output, /##teamcity\[testSuiteFinished name='phantomjs']/);
```

Comment:
I understand that this is currently not testing the order in which these messages are reported, but I guess that might be a good idea to make sure that `testSuiteFinished` is only printed *after* all the tests have finished 🤔

Reply:
I think what you're asking about is tested in the next test.